Xamarin.Forms.Platform.iOS/Renderers/TabbedRenderer.cs

```diff
@@ -28,7 +28,24 @@ public override UIViewController SelectedViewController
 			get { return base.SelectedViewController; }
 			set
 			{
+				if (ReferenceEquals(value, MoreNavigationController))
+				{
+					if(CustomizableViewControllers != null)
+						throw new NotSupportedException("Editing the order of tab items under the More button is not supported.");
```

Comment:
Check against this property because we don't want non-content pages moved to the list.